exercises/all-your-base/canonical-data.json

```diff
@@ -16,169 +16,280 @@
     {
       "description": "single bit one to decimal",
       "property": "rebase",
-      "input_base": 2,
-      "input_digits": [1],
-      "output_base": 10,
+      "input": {
+        "base": 2,
+        "digits": [
+          1
+        ],
+      "output_base": 10
+      },
       "expected": [1]
     },
     {
       "description": "binary to single decimal",
       "property": "rebase",
-      "input_base": 2,
-      "input_digits": [1, 0, 1],
-      "output_base": 10,
+      "input": {
+        "base": 2,
+        "digits": [
+          1,
```

Comment:
Any specific reason to list each digit on a separate line? Seems to be a little much here (and for the other cases too).

Reply:
My IDE automatically formatted it and I figured if that is how JSON is formatted, then I wouldn't object.